spec/system/view_pages_spec.rb

```diff
@@ -0,0 +1,13 @@
+require "rails_helper"
+
+RSpec.feature "View pages", type: :system do
```

Comment:
The capital letter thing I already mentioned.

Reply:
done https://github.com/DFE-Digital/register-trainee-teacher-data/pull/8/commits/c861f6aa8b7f3f8592200da2691fb542a5363e44